kubric/assets/utils.py

```diff
@@ -25,6 +25,32 @@
                    "Sponge", "Spot", "Teapot", "Suzanne")
 
 
+def get_lfn_lights(
+    light_jitter: float = 1.0,
+    rng: np.random.RandomState = randomness.default_rng()):
+  """ Create lights that match the setup from the CLEVR dataset."""
+  sun = core.DirectionalLight(name="sun",
+                              color=color.Color.from_name("white"), shadow_softness=0.2,
+                              intensity=0.85, position=(11.6608, -6.62799, 25.8232))
```

Comment:
Pretty sure the "sun" light is useless in this case because of the enclosing sphere (shines only on the outside).